packages/relative-values/models/src/animals.ts

```diff
@@ -0,0 +1,153 @@
+    {
+      id: "chicken_value_human_qalys_per_lifetime",
+      name: "chicken value per lifetime",
+    },
+    {
+      id: "chicken_value_human_qalys_per_calorie",
+      name: "chicken value per calorie",
+    },
+    {
+      id: "salmon_value_human_qalys_per_year_of_life",
+      name: "salmon value per year of life",
+    },
+    {
+      id: "salmon_value_human_qalys_per_lifetime",
+      name: "salmon value per lifetime",
+    },
+    {
+      id: "salmon_value_human_qalys_per_calorie",
+      name: "salmon value per calorie",
+    },
+    {
+      id: "fish_welfare_initiative_2022_value_estimate",
+      name: "Fish Welfare Initiative (2022)",
+    },
+    {
+      id: "open_wing_alliance_2022_value_estimate",
+      name: "Open Wing Alliance (2022)",
+    },
+    {
+      id: "beyond_meat_2022_value_estimate",
+      name: "Beyond Meat (2022)",
+    },
+    {
+      id: "faunalytics_2022_value_estimate",
+      name: "Faunalytics (2022)",
+    },
+    {
+      id: "the_humane_league_2022_value_estimate",
+      name: "The Humane League (2022)",
+    },
+    {
+      id: "good_food_institute_2022_value_estimate",
+      name: "Good Food Institute (2022)",
+    },
+    {
+      id: "wild_animal_inititative_2022_value_estimate",
+      name: "Wild Animal Inititative (2022)",
+    },
+    {
+      id: "chkd_2022_value_estimate",
+      name: "Çiftlik Hayvanlarını Koruma Derneği (CHKD) (2022)",
+    },
+    {
+      id: "compassion_usa_2022_value_estimate",
+      name: "Compassion USA (2022)",
+    },
+    {
+      id: "dansk_vegetarisk_forening_2022_value_estimate",
+      name: "Dansk Vegetarisk Forening (DVF) (2022)",
+    },
+    {
+      id: "dharma_voices_for_animals_2022_value_estimate",
+      name: "Dharma Voices for Animals (2022)",
+    },
+    {
+      id: "federation_of_indian_animal_protection_organisations_2022_value_estimate",
+      name: "Federation of Indian Animal Protection Organisations (2022)",
+    },
+    {
+      id: "material_innovation_initiative_2022_value_estimate",
+      name: "Material Innovation Initiative (2022)",
+    },
+    {
+      id: "mercy_for_animals_2022_value_estimate",
+      name: "Mercy For Animals (2022)",
+    },
+    {
+      id: "new_harvest_2022_value_estimate",
+      name: "New Harvest (2022)",
+    },
+    {
+      id: "sinergia_animal_2022_value_estimate",
+      name: "Sinergia Animal (2022)",
+    },
+    {
+      id: "xiaobuvegan_2022_value_estimate",
+      name: "xiaobuVEGAN (2022)",
+    },
+  ];
+
+  return {
+    id: "relative-values-animals-2023-04",
+    title: "Relative values for animals and animal charities",
+    items,
+    clusters: {
+      animals: {
+        name: "Animals",
+        color: "#DB828C",
+      },
+    },
+  };
+}
+
+function getTextModel(): Model {
+  return {
+    id: "relative-values-animals-2023-04",
+    title: "Estimates for the relative value of animals and animal charities",
+    author: "Nuno Sempere",
+    mode: "text",
+    code: `
```

Comment:
Reminder that you can use <code>sq``</code> to get embedded syntax highlighting in VS Code. (`sq` is a function that's exported from @quri/squiggle-lang)

Reply:
Oh, I didn't realize, that's neat

Reply:
I did this last month, #1574.